### ISSUE TITLE
Provide instructions for headless systems/servers

### ISSUE THREAD

**Issue by yurikoles** [harababurel/gcsf#69]
In my case I had opened login link on my desktop, but OAuth redirect is failed, because client was running on different machine. Simple `wget -q <redirect-url>` in other terminal window on server machine finished login successfully. 

Please also examine `insync-headleass`, which allows login with auth code, which is generated by Google OAuth flow, from any machine.

**Comment by harababurel** [comment]
I think the issue you're describing is already covered by an option that can be set in [`gcsf.toml`](https://github.com/harababurel/gcsf/blob/f755cc7bafb3dd60af88cd8b8bc26e91efe48888/sample_config.toml#L39-L45):

```
# If set to true, Google Drive will provide a code after logging in and
# authorizing GCSF. This code must be copied and pasted into GCSF in order to
# complete the process. Useful for running GCSF on a remote server.
#
# If set to false, Google Drive will attempt to communicate with GCSF directly.
# This is usually faster and more convenient.
authorize_using_code = false
```

This doesn't involve any redirects and works well for headless setups.

I might have misread your comment, please let me know if this is the case.

**Comment by yurikoles** [comment]
Yes, this is it, sorry for noise. But I still think it should be better documented.
